src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -26,12 +27,10 @@ class FormStartControls extends React.Component {
 
   captureAnalytics = () =>
     this.props.gaStartEventName &&
-    window.dataLayer.push({ event: this.props.gaStartEventName });
```

Comment:
`recordEvent()` is just a wrapper around `window.dataLayer.push()` with a check to make sure that it exists before calling it.